src/systemctl/systemctl.c

```diff
@@ -6307,6 +6308,12 @@ static int log_setting_internal(sd_bus *bus, const BusLocator* bloc, const char
         int r;
 
         if (value) {
+                if (level) {
+                        if (log_level_from_string(value) < 0)
```

Comment:
i'd allow numerical levels too, after all they are standardized pretty much. But either is fine to me. However, the man page in your earlier commit of this PR claims that numbers are accepted too, that part needs to be adjusted hence.

Reply:
OK, let's allow numerical too.

Reply:
Actually, the code was allowing numerical too, so I only needed to adjust the man page ;) I tested that it rejects invalid names and big numbers, but I didn't test single integers ;)